Epicure/src/components/NevigationBar/NevigationBar.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+  StyledIcon,
+  NavigationContainer,
+  LastStyledIcon,
+  RightContainer,
+  CenteredContainer,
+  MenuOption
+} from "./NevigationBarStyles";
+import Logo from '../../assets/Icons/about-logo@3x 1.png';
+import { NavigationBarProps } from './Types';
+
+
+const handleIconClick = (index: number) => {
+    console.log(`Icon ${index} clicked`); 
+  };
```

Comment:
function location should be inside the component instead of being at the top level of it